instantout/fsm.go

```diff
@@ -267,17 +271,25 @@ func (f *FSM) GetV1ReservationStates() fsm.States {
 		},
 		PushPreimage: fsm.State{
 			Transitions: fsm.Transitions{
-				OnSweeplessSweepPublished: WaitForSweeplessSweepConfirmed,
-				fsm.OnError:               Failed,
-				OnErrorPublishHtlc:        PublishHtlc,
-				OnRecover:                 PushPreimage,
+				OnSweeplessSweepBuilt: PublishSweeplessSweep,
```

Comment:
I think it is cleaner if we extract the sweepless sweep tx creation from `PushPreiamgeAction` to `PublishSweeplessSweepAction` and then rename the transition `OnSweeplessSweepBuilt` to `OnPreimagePushed`.
WDYT?